### ISSUE TITLE
&: shortcut doesn't work with custom attributes

### ISSUE THREAD

**Issue by kushniryb** [Netflix/fast_jsonapi#250]
Hey guys 🖖 

First of all - thank you for this awesome gem, great work 👍 
Secondly - I think I may have found a small issue related to the following line - 
https://github.com/Netflix/fast_jsonapi/blob/master/lib/fast_jsonapi/serialization_core.rb#L78

Custom attributes can be defined using following syntax
```ruby
attribute :example do |object|
  object.example?
end
```
Theoretically I should be able to write it as `attribute :example, &:example?` but it doesn't work since `:example?.to_proc.arity #=> -1`

Please let me know what you think about it 🙂

**Comment by shishirmk** [comment]
@kushniryb could you add a test case for this so that we have a good way to look at why it would be failing?

**Comment by ababich** [comment]
I can add here another issue:
`attribute :example, &:computed_example`

where `computed_example` has non-required params like this
```ruby
def computed_example(locale = I18n.default_locale)
  ...
end
```

as a result we will always get issues with `&:` definition

I believe it will work better if by default we will not send `serialization_params` to these methods or maybe introduce condition param for `attribute` to clarify this behavior?

**Comment by Erol** [comment]
@shishirmk Encountered this as well and opened a PR

**Comment by shishirmk** [comment]
This feature is released in [version 1.3](https://github.com/Netflix/fast_jsonapi/releases/tag/1.3)
